dev-python/uri_template/metadata.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>gentoo@chymera.eu</email>
+		<name>Horea Christian</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>python@gentoo.org</email>
+		<name>Python</name>
+	</maintainer>
+	<upstream>
+		<remote-id type="gitlab">gitlab.linss.com/open-source/uri_template/</remote-id>
+	</upstream>
+</pkgmetadata>
```

Comment:
We can probably add `<stabilize-allarches/>` here, since the package is all python and shell scripts.